src/components/Tooltip/Tooltip.tsx

```diff
@@ -104,6 +107,18 @@ export const Tooltip: FC<TooltipProps> = React.memo(
         `${tooltipId?.current}-reference`
       );
 
+      const isMobile: boolean = isIOS() || isAndroid();
+      let mergedTrigger: 'click' | 'hover' | 'contextmenu';
+
+      // First use the most reliable way of detecting iOS or Android,
+      // then test for touch interaction as a somewhat less reliable
+      // way to detect other hybrid devices.
+      if (isMobile || (!isMobile && isTouching())) {
```

Comment:
in theory this now will work in chrome emulation as well?

Reply:
Verified.